.github/workflows/dynamoNet6.0_linux_build.yml

```diff
@@ -0,0 +1,37 @@
+# Build Dynamo using latest VS and DotNET
+name: Dynamo-net6.0-linux-Build
+on: [push,pull_request]
+jobs:
+ build:
+  runs-on: ubuntu-latest
+  steps:
+    - name: Checkout Dynamo Repo
+      uses: actions/checkout@v2
+      with:
+        path: Dynamo
+        repository: mjkkirschner/Dynamo
```

Comment:
?

Reply:
nice catch - that let me test this github action against *this* code - but it will be wrong when I merge it.